lib/webdriverLogger.ts

```diff
@@ -0,0 +1,55 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as stream from 'stream';
+
+import {CommandName, WebDriverCommand} from './webdriverCommands';
+
+export class WebDriverLogger {
+  logStream: stream.Writable;
+
+  public setLogDir(logDir: string) {
+    let logName = `webdriver_log_${process.pid}.txt`;
```

Comment:
I wonder if we should use a timestamp instead of process.pid here - pid might get reused, and it's non-obvious which one is the latest if you've run a bunch of instances.